templates/shared_services/firewall/porter.yaml

```diff
@@ -49,6 +49,8 @@ parameters:
     type: string
     default: Standard
     description: The firewall and its policy SKU tier
+  - name: microsoft_graph_fqdn
```

Comment:
I wonder if we should add a default for all these types of params that will represent the value in the public cloud. This way we could allow easier install in the primary cloud a developer is likely to use.

Reply:
Done